README.md

```diff
@@ -53,6 +53,7 @@ data into vector tiles that can be rendered dynamically.
 - [vt-geojson](https://github.com/developmentseed/vt-geojson) - decodes vector tiles to GeoJSON FeatureCollections
 - [tl](https://github.com/mojodna/tl) - An alternate command line interface to tilelive
 - [tileshrink](https://github.com/rastapasta/tileshrink) - Reduce the layer extent and simplify the resulting geometries of all vector tiles in an MBTiles
+- [tiler](https://github.com/Geovation/tiler) - A no nonsense Vector Tile pipeline
```

Comment:
"no nonsense" is marketing-speak - please follow the other examples and keep this specific and dry.